modules/spotxBidAdapter.js

```diff
@@ -0,0 +1,417 @@
+import * as utils from 'src/utils';
+import { Renderer } from 'src/Renderer';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { VIDEO } from 'src/mediaTypes';
```

Comment:
Imports like these need to be relative after `1.38.0`

See: 

so add `../` before each `src`